cmd/ocsp-updater/main.go

```diff
@@ -586,14 +588,32 @@ func setupClients(c cmd.OCSPUpdaterConfig, stats metrics.Scope) (
 	core.StorageAuthority,
 ) {
 	amqpConf := c.AMQP
-	cac, err := rpc.NewCertificateAuthorityClient(clientName, amqpConf, stats)
-	cmd.FailOnError(err, "Unable to create CA client")
+
+	var cac core.CertificateAuthority
+	if c.SAService != nil {
```

Comment:
I don't think it merits a separate PR, just wanted to be sure I understood. Thanks! :cake: